.github/scripts/run-docker.sh

```diff
@@ -1,17 +1,16 @@
 #!/bin/sh
 
-set -euo pipefail
+set -eo pipefail
 
 # Version of the config
 CONFIG_VER=v0
 
 # Run Parameters
 RUN_SCRIPT=$1
-CONFIG_DIR=/workspace/benchmark/score/configs/${CONFIG_VER}
+CONFIG_DIR=${PWD}/score/configs/${CONFIG_VER}
```

Comment:
is this path wrong now? i moved score into torchbenchmark/score/...

Reply:
Right. Thanks for pointing out. Updated.